src/gui/tray/Window.qml

```diff
@@ -99,7 +99,9 @@ Window {
         anchors.fill:   parent
         radius:         Style.trayWindowRadius
         border.width:   Style.trayWindowBorderWidth
-        border.color:   Style.menuBorder
+        border.color:   Style.windowText

```

Comment:
@FlexW You sure this is not `Style.window`?

Reply:
Yes I'm sure. It was like that before. However, I'm not sure it should be better button.

Reply:
I will schedule a call with jan for that;)